src/main/histFileMod.F90

```diff
@@ -301,10 +301,22 @@ subroutine hist_printflds()
     ! !DESCRIPTION:
     ! Print summary of master field list.
     !
+    ! !USES:
+    use clm_varctl, only: master_list_file, hist_master_list_file
+    !
     ! !ARGUMENTS:
     !
     ! !LOCAL VARIABLES:
-    integer nf
+    integer nf, i  ! do-loop counters
+    integer width_col_1  ! widths of table columns 1...
+    integer width_col_2  ! ...2
+    integer width_col_3  ! ...3
+    integer width_col_4  ! ...and 4
+    integer width_col_sum  ! widths of columns summed, including spaces
+    character(len=1) str_w_col_1  ! string versions of width_col_1 through 4
+    character(len=2) str_w_col_2, str_w_col_3, str_w_col_4
+    character(len=3) str_w_col_sum  ! string version of width_col_sum
+    character(len=25) fmt_txt  ! format statement
```

Comment:
- [ ] [optional] I'd suggest making these string lengths longer so that this doesn't break under modification. (If need be, you can then `trim` them where they are used, but I have a feeling that isn't necessary.)

Reply:
Changing to lengths of 3,3,3,99. For the first three, I limit the lengths to 3 characters because I want to discourage columns wider than 999.

I tested without adding `trim` and got identical results.

Reply:
Makes sense - thanks.